frontend/website/src/Developers.re

```diff
@@ -0,0 +1,160 @@
+let extraHeaders = () =>
+  <>
+    <script src="https://apis.google.com/js/api.js" />
```

Comment:
do we need either of these? I don't think we'll be using gapi or marked on this page??